src/typeahead/typeahead.js

```diff
@@ -170,6 +169,28 @@ angular.module('ui.bootstrap.typeahead', ['ui.bootstrap.position', 'ui.bootstrap
         });
       };
 
+      // bind events only if appendToBody params exist - performance feature
+      if ( appendToBody ) {
+        angular.element( $window ).bind( 'resize', fireRecalculating );
+        $document.find( 'body' ).bind( 'scroll', fireRecalculating );
```

Comment:
Both these event listeners need to be throttled - the resize and scroll events both fire often.